packages/grafana-toolkit/docker/grafana-plugin-ci/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM circleci/node:12-browsers
+USER root 
+WORKDIR /tmp
+
+# Install Go
+ADD https://dl.google.com/go/go1.14.linux-amd64.tar.gz /tmp
+RUN echo 08df79b46b0adf498ea9f320a0f23d6ec59e9003660b4c9c1ce8e5e2c6f823ca go1.14.linux-amd64.tar.gz | sha256sum --check --status
+RUN sudo tar -C /usr/local -xf go1.14.linux-amd64.tar.gz
+
+# Install golangci-lint
+ADD https://github.com/golangci/golangci-lint/releases/download/v1.23.7/golangci-lint-1.23.7-linux-amd64.tar.gz /tmp
+RUN echo 34df1794a2ea8e168b3c98eed3cc0f3e13ed4cba735e4e40ef141df5c41bc086 golangci-lint-1.23.7-linux-amd64.tar.gz | sha256sum --check --status
+RUN tar xf golangci-lint-1.23.7-linux-amd64.tar.gz
+RUN sudo mv golangci-lint-1.23.7-linux-amd64/golangci-lint /usr/local/bin
+RUN sudo ln -s /usr/local/go/bin/go /usr/local/bin/go
+RUN sudo ln -s /usr/local/go/bin/gofmt /usr/local/bin/gofmt
+
+# Install dependencies
+RUN sudo apt-get install -y adduser libfontconfig1
+
+# Install code climate
+ADD https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 /usr/local/bin/cc-test-reporter
+RUN chmod +x /usr/local/bin/cc-test-reporter
+
+# Download, but don't install previous grafana releases
+RUN mkdir -pv /usr/local/grafana/deb
+ADD https://dl.grafana.com/oss/release/grafana_6.6.2_amd64.deb /usr/local/grafana/deb
+ADD https://dl.grafana.com/oss/release/grafana_6.5.3_amd64.deb /usr/local/grafana/deb
+ADD https://dl.grafana.com/oss/release/grafana_6.4.5_amd64.deb /usr/local/grafana/deb
+ADD https://dl.grafana.com/oss/release/grafana_6.3.7_amd64.deb /usr/local/grafana/deb
+
+# Perform user specific initialization
+USER circleci
+RUN mkdir -pv ${HOME}/plugin ${HOME}/go/bin ${HOME}/bin ${HOME}/src ${HOME}/tmp
+
+# Install grafana release with yarn
+RUN git clone https://github.com/grafana/grafana.git ${HOME}/src/grafana
```

Comment:
next should it checkout the 'latest' branch?
https://github.com/grafana/grafana/blob/master/latest.json

>> v6.6.2?